Assignment 3.Rmd

```diff
@@ -136,18 +160,76 @@ What patterns do you see in the plot?
 It would be useful to determine how many people are in each cluster. We can do this easily with dplyr.
 
 ```{r}
-K7 <- 
+K7 <- dplyr::count(K4, Cluster)
+#8 People in Cluster 1, 15 people in Cluster 2
+K72 <- dplyr::count(K42, Cluster)
+#7 people in Cluster 1, 9 people in Cluster 2, 7 peope in Cluster 3.
 ```
 
 Look at the number of people in each cluster, now repeat this process for 3 rather than 2 clusters. Which cluster grouping do you think is more informative? Write your answer below:
 
+The 3 Kmeans cluster shows slightly more information. 
+
```

Comment:
What do you mean by "more information"? What is the information you are trying to display?